### DISCUSSION TITLE
Long rclone push times with .git

### DISCUSSION THREAD

**Discussion by ToasterSticks** [coder/code-server#3153] [Q&A]
Hey there, I'm experiencing times up to an hour with my .git/objects/... files. Is there a way I can zip my "Projects" folder that contains all my folders before uploading it to GDrive, and when pulling, it would unzip the zipped `Projects.zip` If so, how would I modify the files, and necessary environment variables. I'm on railway by the way.

**Comment by bpmct** [comment]
Hey @ToastOnAStik.

Yeah we could add something in that does that, but was curious how this would be helpful. I remember you were earlier asking how to exclude the git folder: https://github.com/cdr/code-server/discussions/3090

Is there a reason you'd like to sync that up?

Ben

**Reply by ToasterSticks** [reply]
Ah sorry, usually I would like to keep the git folders as they are only a few mb. It might be the sheer amount of files inside of it.

**Reply by bpmct** [reply]
That makes sense. If I'm understanding correctly, it should zip everything up? That seems pretty do-able

**Reply by ToasterSticks** [reply]
Yep
